webapp/src/components/KeyValueLi/ValueLi/index.tsx

```diff
@@ -39,7 +39,7 @@ const ValueLi: React.FunctionComponent<KeyValueLiProps> = (
           {props.value ? (
             <EllipsisText
               text={props.value}
-              length={Number(props.textLimit ?? `50`)}
+              length={new BigNumber(props.textLimit ?? `50`).toNumber()}
```

Comment:
this doesn't need big number, please remove this conversion, BigNumber is for account balances